crosstool/universal_exec_tool.bzl

```diff
@@ -0,0 +1,30 @@
+def universal_exec_tool(*, name, out, srcs):
+    native.genrule(
+        name = name,
```

Comment:
I think you actually want apple_genrule so we get the Xcode variables?

Reply:
Pretty sure that makes a circular reference. I'll try it though.

Reply:
Hmm, it worked, sweet!